docker-java-api/src/main/java/com/github/dockerjava/api/model/ExposedPorts.java

```diff
@@ -35,7 +35,9 @@ public static ExposedPorts fromPrimitive(Map<String, Object> object) {
 
     @JsonValue
     public Map<String, Object> toPrimitive() {
-        return Stream.of(exposedPorts).collect(Collectors.toMap(
+        return Stream.of(exposedPorts)
+            .distinct()
```

Comment:
instead of `distinct()`, we can use a merge strategy (3rd argument of `toMap`) that does not throw, but uses the previously set value